examples/stocks/lib/stock_home.dart

```diff
@@ -24,15 +24,16 @@ class StockHomeState extends State<StockHome> {
   final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
   bool _isSearching = false;
   String _searchQuery;
-  TabBarSelection _tabBarSelection;
+  int _initialSelectedTabIndex;
 
   void initState() {
     super.initState();
-    _tabBarSelection = PageStorage.of(context)?.readState(context);
-    if (_tabBarSelection == null) {
-      _tabBarSelection = new TabBarSelection(maxIndex: 1);
-      PageStorage.of(context)?.writeState(context, _tabBarSelection);
-    }
+    _initialSelectedTabIndex = PageStorage.of(context)?.readState(context) ?? 0;
+  }
+
+  void dispose() {
+    super.dispose();
```

Comment:
nit: good practice puts the super.dispose() call second, on the principle that it might otherwise dispose stuff that this method relies on. (It doesn't, but...)